### DISCUSSION TITLE
Diploma VS Portifólio.

### DISCUSSION THREAD

**Discussion by marcusvcribeiro** [frontendbr/forum#509] [Discussão]
O que vale mais, um Diploma ou um portfólio???

Hoje moro em brasília, um local que infelizmente ainda muitas das empresas ainda olham currículo ao invés de portfólio, devido a quantidade de empresas que tem seus contratos com o Governo.

Ai eu me pergunto, o que é um currículo perto de um portfólio?
Porque o que me soa quando eu vejo uma pessoa que saiu da faculdade sem currículo nenhum, é:

Fui para faculdade todos os dias, decorei um monte de coisa e terminei o curso.

Agora quando você vê o portfólio de alguém, ali você consegue "ver" o conhecimento da pessoa.
Esses dias vi uma citação que me fez pensar bastante sobre isso,

"O mundo vai valorizar cada vez menos diplomas e certificados e cada vez mais o que você, de fato, tem criado com o conhecimento adquirido, ou seja: seu portfólio." por Mathieu Le Roux

Na opinião de vocês, o que vale mais? Quais os pontos positivos de ter uma faculdade? Quais os pontos negativos de não ter?

**Comment by paoloalmeida** [comment]
Rapaz, muito boa a pergunta!

Ansioso pelas respostas kk... mas vai aqui minha singela opinião:

Nunca fui barrado em vagas, por não ter diploma, ao invés disso sempre mantive meu linkedin/portifolio em dia.... Até porquê diploma não garante muita coisa ao contratante, na minha opinião. Afinal, o que ele terá que entregar ao cliente é o job, e em alguns casos, recém formados podem saber muito da teoria, mas não da prática.
Hoje como meu objetivo é sair do Brasil, a facul entrou na minha lista de *get shit done*, mas só pra ajudar no visto.

Abs!

**Comment by familiatsilva** [comment]
Marcus, 

Trabalho em uma editora aqui em são paulo e nos ultimos meses entrevistei mais de 30 pessoas. Em nenhum momento pedi para apresentarem o diploma ou comprovarem que estavam fazendo faculdade. O que mais contou para mim eram suas experiências profissionais e como elas conseguiriam me ajudar a tocar os projetos que cuido.

Acredito que a faculdade te dará uma pequena noção do que é o mercado de trabalho, quais as tecnologias estão sendo utilizadas por ai e quais as ramificações possíveis um graduando pode seguir. A faculdade serve mais como um nivelador, tentando deixar todos no mesmo patamar. Infelizmente muitas faculdades hoje não preparam o candidato, ele sai mais perdido do que entrou...

Para mim o que mais importa são os projetos desenvolvidos, as participações em times pequenos ou grandes e responsabilidades assumidas. Um jovem sai da faculdade com o minimo de conhecimento, e será no dia-dia que ele poderá lapidar o que aprendeu e avançar em sua carreira.

Um portfolio bem feito e um curriculo bem detalhado me ajuda a ter uma noção de onde o candidato passou, quais dificuldades encontrou e como poderá me ajudar em minha empresa.

Só que ai entra um outro ponto... Muitos candidatos tem mentido no curriculo e no portfolio. Quando investigo com mais profundidade dizem que apenas ajudou numa telinha aqui ou ali. Outros colocam diversas coisas no curriculo se dizendo azes dessa ou daquela tecnologia, porém quando coloco na parede ou aplico teste pratico, só sabem a teoria.. Está faltando mais veracidade nos curriculos e portfolios.

Resumindo: Para mim vale mais um curriculo recheado de experiencias do que um diploma lindo da universidade de harvard

**Comment by ericdouglas** [comment]
Enquanto não houver uma regularização da profissão de desenvolvimento de software como existe em áreas como Engenharia, Advocacia, etc, onde apenas profissionais com registro em um conselho podem exercer determinada profissão, o foco/atenção do profissional deve ser:

**Conhecimento vs Falta de conhecimento**

e não Diploma vs Portfólio

Faculdade é um meio de se obter conhecimento, estudo autodidata é outra forma de se obter conhecimento. 

Com base nos seus objetivos profissionais, você deve otimizar seu tempo para obter o maior nível de conhecimento **útil** no menor espaço de tempo.

Você consegue ter o conhecimento de um "diplomado" e também criar um portfólio seguindo um programa [desse tipo](https://github.com/open-source-society/computer-science). Uma coisa não exclui a outra, você pode perfeitamente ter diploma E portfólio!

> Quais os pontos positivos de ter uma faculdade?

Um ponto positivo de se ter um diploma é a maior possibilidade de conseguir um visto de trabalho em outros países. Caso a pessoa tenha esse sonho, é altamente recomendado que ela tenha um curso superior.

> Quais os pontos negativos de não ter?

O que realmente penaliza um profissional nessa área é não ter **conhecimento**. Não ter diploma não afeta em nada, pois você pode obter conhecimento de outras formas.

Uma empresa que elimine um candidato com muito conhecimento mas sem diploma não vale a pena de se trabalhar IMO, e eu descartaria a empresa primeiro hehe.

**Reply by renanlido** [reply]
Parabéns! Em 2021 este comentário continua super atual!

**Comment by felipefialho** [comment]
Linkando:

#39

**Comment by willycamargo** [comment]
Nem sei se seria melhor eu postar isso aqui ou na issue #39 , mas talvez seja porque o artigo sirva para as duas =)

[Eric Elliott: Want to Code? A University Degree Might be a Huge Waste of Time](https://medium.com/javascript-scene/want-to-code-a-university-degree-might-be-a-huge-waste-of-time-81e1817a2ef0)

**Comment by rubenmarcus** [comment]
Depende da situação o Diploma ajuda como na Issue , sobre Trabalhar no Exterior..
E dependendo da vaga, ou freela, mais que conhecimento,portfólio ou diploma, o QI , também conta muito.

Mas falando de um experiência aqui, parecida com a sua.. Ano passado estava vendo vagas na ONU, para front.. Todas pediam Ensino Superior.. E eu não tenho. E me inscrevi mesmo assim. E fui selecionado para fase de entrevistas.. Era um trabalho como consultor...  Mas não me chamaram e nem deram feedback. Mas pra mim ja valeu a pena, ver que me consideraram apenas pelas skills e portfolio,mesmo sem diploma.

Acho que trabalhar na sua cidade é um bom aprendizado e segurança,mas temos que pensar no mercado de TI como algo global e nos compararmos com profissionais de fora também...

**Comment by ericdouglas** [comment]
@rubenmarcus você falou um ponto importantíssimo, o QI (Quem Indica)! Muito bem lembrado :smile:

Participar de meetups, projetos open-source, escrever artigos, tudo isso pode ser mais decisivo do que as primeiras opções que discutimos.

**Comment by didiraja** [comment]
Diploma não deveria mas influencia, pelos mais variados motivos. Seja conservadorismo ou contratante desonesto que só aceita te pagar a mais se tiver diploma (ouvi muito essa história na época de faculdade).

Na dúvida, tenha diploma. Nenhum desenvolvedor morre de fome por falta de diploma, mas tê-lo te evita a perder algumas oportunidades.

**Comment by bernardodiasc** [comment]
Já perdi oportunidades por não ter diploma, e perder algumas oportunidades significou ganhar outras 😉 💪

**Comment by marabesi** [comment]
Diploma não é sinal de nada, mas na minha opinião se você não tiver irá perder várias oportunidades.

A primeira que se vai é a possibilidade de trabalhar no exterior(a não ser que você vá sozinho ou tenha quem te indique). A segunda é em pré-requisitos das vagas, muitas vagas pedem no mínimo cursando um curso de graduação.

A terceira e mais importante pra mim é a experiência e networking que você pode fazer na faculdade. Mesmo que a faculdade não seja boa ou seus colegas não estão nem ai pro curso, você  ja sabe a quem não seguir e se esforçar ao máximo para não ser como um deles : )

No final eu não vejo um Diploma vs Portifólio, e sim um Diploma + portifólio. Os dois possuem pesos importantes e devem ser considerados.

**Comment by lucasjs** [comment]
Vocês querem dizer que o diploma e sua função servem de nada no caso de um péssimo aluno e péssima instituição de ensino né? Espero que seja isso, porque falar que o ensino superior serve pra nada é tenso ein.

**Comment by nighto** [comment]
Entrar na faculdade (veja bem, entrar, não sair) foi interessante para construir uma rede de contatos na década passada. Grande parte dos estágios e depois empregos que trabalhei - inclusive o atual - foram indicações de ex-colegas de faculdade.

Sempre fui pró-portifolio e anti-faculdade, mas 10 anos depois de entrar (e não diplomar...) está fazendo falta para trabalhar no exterior. Nem todo país exige, mas alguns sim. Já fui barrado em duas oportunidades aonde o entrevistador me disse que eu atendia os requisitos técnicos, mas por razões burocráticas do país em que a empresa se encontrava, seria necessário diploma para emitir o visto de trabalho. Por este motivo estou correndo atrás, fazendo Cederj (tecnólogo semi-presencial, o que significa aulas online e provas presenciais).

**Comment by felipefialho** [comment]
Esperei eu iniciar aqui no @cubonetwork para dar minha opinião sobre o assunto.

Desde sempre fui da opinião da vantagem absurda de portfolio sobre diploma. E continuo mantendo ela.

Porém também comentava que não via mais necessidade de ter um diploma no cenário de desenvolvimento atual. E nesse caso, mudei de opinião.

Essa vaga só foi possível em razão do diploma, e várias e várias outras empresas super legais, acabam não permitindo que novos funcionários ingressem caso não tenham feito faculdade. Se você concorda ou não com elas é outra história, mas é assim que o mercado funciona.

Outra questão é com relação as vagas no exterior e vistos, comprovadamente eles são facilitados caso você tenha curso superior.

Ou seja, deixe seu portfolio do carario e foque nele mais do que tudo. Mas caso ainda não tenha, vá atrás de um curso superior na área.

**Comment by fabioassuncao** [comment]
Cara, o diploma é importante sim e tem seu valor, mas é inegável que experiência/portfólio tenha um peso na contratação e em muitos casos, seja fator determinante. Eu comecei a cursar Ciências da Computação e ainda não terminei (n motivos)... Mesmo tendo experiência, ter trabalhado em algumas empresas (presencial e remoto), devo admitir que a falta de um diploma de curso superior já me fez passar por algumas frustrações e até perder oportunidades.

**Comment by wilsonneto-swe** [comment]
Seu diploma e seu portfólio são duas coisas diferentes, a avaliação se feita criteriosamente e de maneira séria vai levar em conta o CHA:

C = Conhecimento (Estudos, Diplomas, Certificações, Aprimoramentos, MBAs... formais e informais)

H = Habilidades (Experiência no trabalho propriamente dito, pratica em lidar com os problemas que aparecem, projetos pessoais, portifolio e etc)

A = Atitude (tem muito cara que é fenomenal nos dois acima, mas não puxa a responsa, não é proativo, e etc, atitude não da para treinar então é mais raro, os dois acima qualquer pessoa que queira e pague o preço consegue)

Ao menos é o que mais vejo nas avaliações sérias de RH e empresas grandes. 
E inclusive a competência, que no final do dia é o que realmente importa para sua carreira, para a empresa na qual trabalha e para o cliente, é exatamente isso, o C.H.A...

Minha opinião, abraços!

**Reply by profmarcioassis** [reply]
Concordo plenamente.
Não vejo diploma e portfólio como coisas excludentes. 
Por que não pensar em construir ambos? Não seria possível?
Sou professor universitário e sempre atuei no mercado. É possível construir um bom portfólio, inclusive durante a faculdade, apesar dela não ser necessária para isso. 

O que mais recebo aqui é: "vale a pena fazer faculdade?"
Isso vai depender do objetivo de cada um. 
Quais habilidades você está buscando?
Faculdade não é só habilidades técnicas. É um lugar de construção de network, de lidar com frustrações, de saber lidar com diferentes pontos de vista, trabalhar em equipe, etc.

Se você busca só conhecimento técnico, pula fora e vai fazer um treinamento.
Caso contrário, vale muito a pena e auxilia muito construir o CHA.
